tests/app/test_email_limit_utils.py

```diff
@@ -19,28 +19,21 @@ def test_fetch_todays_requested_email_count(self, client, mocker, sample_service
 
             assert actual_result == expected_result
             if redis_value is None:
-                assert mocked_set.called_once_with(
+                mocked_set.assert_called_once_with(
                     cache_key,
                     db_value,
+                    ex=7200,
                 )
             else:
                 mocked_set.assert_not_called()
 
     @pytest.mark.parametrize("redis_value, db_value, increment_by", [(None, 5, 5), ("3", 5, 3)])
-    def test_increment_todays_requested_email_count(self, mocker, sample_service, redis_value, db_value, increment_by):
+    def test_increment_todays_requested_email_count(self, client, mocker, sample_service, redis_value, db_value, increment_by):
         cache_key = email_daily_count_cache_key(sample_service.id)
         mocker.patch("app.redis_store.get", lambda x: redis_value if x == cache_key else None)
-        mocked_set = mocker.patch("app.redis_store.set")
         mocked_incrby = mocker.patch("app.redis_store.incrby")
         mocker.patch("app.email_limit_utils.fetch_todays_email_count", return_value=db_value)
 
-        increment_todays_email_count(sample_service.id, increment_by)
-
-        assert mocked_incrby.called_once_with(cache_key, increment_by)
-        if redis_value is None:
-            assert mocked_set.called_once_with(
-                cache_key,
-                db_value,
-            )
-        else:
-            mocked_set.assert_not_called()
+        with set_config(client.application, "REDIS_ENABLED", True):
```

Comment:
If we don't set `REDIS_ENABLED` then `increment_todays_email_count()` immediately returns `None`, and it turns out that `REDIS_ENABLED` is false in CI by default.